SWLOR.Game.Server/ChatCommand/PerkResetCooldown.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Linq;
+using NWN;
+using SWLOR.Game.Server.ChatCommand.Contracts;
+using SWLOR.Game.Server.Data.Entity;
+using SWLOR.Game.Server.Enumeration;
+using SWLOR.Game.Server.GameObject;
+using SWLOR.Game.Server.Service;
+using static NWN._;
+
+namespace SWLOR.Game.Server.ChatCommand
+{
+    [CommandDetails("Resets a player's perk refund cooldowns.", CommandPermissionType.DM | CommandPermissionType.Admin)]
+    public class PerkResetCooldown : IChatCommand
+    {
+        public void DoAction(NWPlayer user, NWObject target, NWLocation targetLocation, params string[] args)
+        {
+            if (!target.IsPlayer)
+            {
+                user.SendMessage("Only player characters may be targeted with this command.");
+                return;
+            }
+
+            var now = DateTime.UtcNow;
+            Player dbPlayer = DataService.Player.GetByID(target.GlobalID);
+            dbPlayer.DatePerkRefundAvailable = null;
```

Comment:
I'd prefer for this to be set to DateTime.UtcNow instead of null.

Reply:
Ahh, is that how you call current time... Okay lemme change that.  (I tried to do that but kept getting issues and Null works XD)